fix: update docker compose and go-ftw packages install (#167)
coreruleset/documentation: content/deployment/install.md

## Alternative: Using Containers

> Another quick option is to use the official CRS [pre-packaged containers]({{% ref "../development/useful_tools/#official-crs-maintained-docker-images" %}}). Docker, Podman, or any compatible container engine can be used. The official CRS images are published in the [Docker Hub](https://hub.docker.com/r/owasp/modsecurity-crs/) and [GitHub Container Repository](https://github.com/coreruleset/modsecurity-crs-docker/pkgs/container/modsecurity-crs). The image most often deployed is `owasp/modsecurity-crs` (from dockerhub) or `ghcr.io/coreruleset/modsecurity-crs`: it already has everything needed to get up and running quickly.

Reviewer: ```suggestion
Another quick option is to use the official CRS [pre-packaged containers]({{% ref "../development/useful_tools/#official-crs-maintained-docker-images" %}}). Docker, Podman, or any compatible container engine can be used. The official CRS images are published on [Docker Hub](https://hub.docker.com/r/owasp/modsecurity-crs/) and [GitHub Container Repository](https://github.com/coreruleset/modsecurity-crs-docker/pkgs/container/modsecurity-crs). The image most often deployed is `modsecurity-crs` (`owasp/modsecurity-crs` from Docker Hub or `ghcr.io/coreruleset/modsecurity-crs` from GHCR): it already has everything needed to get up and running quickly.
```

The CRS project pre-packages both Apache and Nginx web servers along with the appropriate corresponding ModSecurity engine. More engines, like [Coraza](https://coraza.io/), will be added at a later date.

To protect a running web server, all that's required is to get the appropriate image and set its configuration variables to make the WAF receives requests and proxies them to your backend server.

> Below is an example docker _compose_ file that can be used to pull the container images. If you don't have compose installed, check the install [here](https://docs.docker.com/compose/install/). All that needs to be changed is the `BACKEND` variable so that the WAF points to the backend server in question:

Reviewer: ```suggestion
Below is an example docker _compose_ file that can be used to pull the container images. If you don't have compose installed, please read the [installation instructions](https://docs.docker.com/compose/install/). All that needs to be changed is the `BACKEND` variable so that the WAF points to the backend server in question:
```
